Fix another flaky integration test

See https://circleci.com/gh/neuromation/platform-storage-api/236?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

```
______________ TestLocalFileSystem.test_rename_dir_different_dir _______________

self = <tests.unit.test_fs_local.TestLocalFileSystem object at 0x7f97d11ca898>
fs = <platform_storage_api.fs.local.LocalFileSystem object at 0x7f97d11caf28>
tmp_dir_path = PosixPath('/tmp/tmpig4_sy02')

    @pytest.mark.asyncio
    async def test_rename_dir_different_dir(self, fs, tmp_dir_path):
        file_name = PurePath("file")
        old_dir = PurePath("old")
        new_dir = PurePath("new")
        nested_dir = PurePath("nested")
        payload = b"test"
        old_path = tmp_dir_path / old_dir
        nested_path = tmp_dir_path / nested_dir
        new_path = nested_path / new_dir
        old_file_path = old_path / file_name
        new_file_path = new_path / file_name
    
        await fs.mkdir(old_path)
        await fs.mkdir(nested_path)
    
        async with fs.open(old_file_path, mode="wb") as f:
            await f.write(payload)
            await f.flush()
    
        old_statuses = set(await fs.liststatus(tmp_dir_path))
        old_statuses_old_dir = set(await fs.liststatus(old_path))
        await fs.rename(old_path, new_path)
    
        statuses = set(await fs.liststatus(tmp_dir_path))
>       assert statuses == self.statuses_drop(old_statuses, old_dir)
E       AssertionError: assert {FileStatus(p...EAD: 'read'>)} == {FileStatus(pa...EAD: 'read'>)}
E         Extra items in the left set:
E         FileStatus(path=PurePosixPath('nested'), type=<FileStatusType.DIRECTORY: 'DIRECTORY'>, size=0, modification_time=1542192195, permission=<FileStatusPermission.READ: 'read'>)
E         Extra items in the right set:
E         FileStatus(path=PurePosixPath('nested'), type=<FileStatusType.DIRECTORY: 'DIRECTORY'>, size=0, modification_time=1542192194, permission=<FileStatusPermission.READ: 'read'>)
E         Full diff:
E         - {FileStatus(path=PurePosixPath('nested'), type=<FileStatusType.DIRECTORY: 'DIRECTORY'>, size=0, modification_time=1542192195, permission=<FileStatusPermission.READ: 'read'>)}
E         ?                                                                                                                            ^
E         + {FileStatus(path=PurePosixPath('nested'), type=<FileStatusType.DIRECTORY: 'DIRECTORY'>, size=0, modification_time=1542192194, permission=<FileStatusPermission.READ: 'read'>)}
E         ?                                                                                                                            ^

tests/unit/test_fs_local.py:586: AssertionError
```

--- a/platform_storage_api/api.py
+++ b/platform_storage_api/api.py
@@ -7,11 +7,12 @@ from typing import Iterator, List, Optional
 import aiohttp.web
 from aiohttp.web_exceptions import HTTPBadRequest, HTTPUnauthorized
 from aiohttp.web_request import Request
-from aiohttp_security import check_authorized, check_permission
-from async_exit_stack import AsyncExitStack
 from neuro_auth_client import AuthClient, Permission, User
 from neuro_auth_client.client import ClientSubTreeViewRoot
 from neuro_auth_client.security import AuthScheme, setup_security
+
+from aiohttp_security import check_authorized, check_permission
+from async_exit_stack import AsyncExitStack
 
 from .config import Config
 from .fs.local import FileStatus, FileStatusPermission, LocalFileSystem

--- a/tests/unit/test_fs_local.py
+++ b/tests/unit/test_fs_local.py
@@ -388,10 +388,12 @@ class TestLocalFileSystem:
 
         statuses = set(await fs.liststatus(tmp_dir_path))
         new_subdir_status = self.statuses_get(statuses, subdir)
-        assert new_subdir_status.modification_time >= \
-            old_subdir_status.modification_time
-        assert statuses | {renaming_status} | {old_subdir_status} == \
-            old_statuses | {new_subdir_status}
+        assert (
+            new_subdir_status.modification_time >= old_subdir_status.modification_time
+        )
+        assert statuses | {renaming_status} | {old_subdir_status} == old_statuses | {
+            new_subdir_status
+        }
 
         statuses = set(await fs.liststatus(subdir_path))
         assert statuses == self.statuses_rename({renaming_status}, old_name, new_name)
@@ -591,10 +593,12 @@ class TestLocalFileSystem:
 
         statuses = set(await fs.liststatus(tmp_dir_path))
         new_subdir_status = self.statuses_get(statuses, nested_dir)
-        assert new_subdir_status.modification_time >= \
-            old_subdir_status.modification_time
-        assert statuses | {renaming_status} | {old_subdir_status} == \
-            old_statuses | {new_subdir_status}
+        assert (
+            new_subdir_status.modification_time >= old_subdir_status.modification_time
+        )
+        assert statuses | {renaming_status} | {old_subdir_status} == old_statuses | {
+            new_subdir_status
+        }
 
         statuses = set(await fs.liststatus(nested_path))
         expected_statuses = self.statuses_drop(old_statuses, nested_dir)

